internal/commands/util/remediation.go

```diff
@@ -225,7 +226,11 @@ func runKicsRemediation(cmd *cobra.Command, volumeMap, tempDir string) error {
 	}
 	if len(kicsSimilarityFilter) > 0 {
 		kicsRunArgs = append(kicsRunArgs, kicsIncludeIdsFlag)
-		kicsRunArgs = append(kicsRunArgs, kicsSimilarityFilter...)
+		filteredIds := ""
+		for _, filterIdElement := range kicsSimilarityFilter {
```

Comment:
kicsSimilarityFilter is a slice? if yes, could we use strings.Join instead of manually looping?

Reply:
yes, it will look better. Done, thanks !